runelite-client/src/main/java/net/runelite/client/plugins/grandexchange/GrandExchangeSearchPanel.java

```diff
@@ -193,7 +201,7 @@ private void processResult(List<ItemPrice> result, String lookup, boolean exactM
 		for (ItemPrice item : result)
 		{
 			if (count++ > MAX_SEARCH_ITEMS)
-			{
+				{
```

Comment:
You should probably set the format settings of your client with the code style guidelines.

https://github.com/runelite/runelite/wiki/Code-Conventions